field/arena_test.go

```diff
@@ -1033,326 +955,3 @@ func TestPlcMatchCycleEvergreen(t *testing.T) {
-			game.MatchTiming.TeleopDurationSec,
-	) * time.Second
-	arena.MatchStartTime = time.Now().Add(-durationToTeleopEnd + 1*time.Millisecond)
-	arena.Update()
-	assert.Equal(t, [3]bool{true, false, false}, plc.redTrussLights)
-	assert.Equal(t, [3]bool{true, true, false}, plc.blueTrussLights)
-	arena.MatchStartTime = time.Now().Add(-durationToTeleopEnd - 1*time.Millisecond)
-	arena.Update()
-	assert.Equal(t, [3]bool{true, true, true}, plc.redTrussLights)
-	assert.Equal(t, [3]bool{true, true, true}, plc.blueTrussLights)
-	arena.MatchStartTime = time.Now().Add(-durationToTeleopEnd - 2999*time.Millisecond)
-	arena.Update()
-	assert.Equal(t, [3]bool{true, true, true}, plc.redTrussLights)
-	assert.Equal(t, [3]bool{true, true, true}, plc.blueTrussLights)
-	arena.MatchStartTime = time.Now().Add(-durationToTeleopEnd - 3001*time.Millisecond)
-	arena.Update()
-	assert.Equal(t, [3]bool{false, false, false}, plc.redTrussLights)
-	assert.Equal(t, [3]bool{false, false, false}, plc.blueTrussLights)
-}
-
-func TestPlcMatchCycleGameSpecificWithCoopDisabled(t *testing.T) {
-	defer func() {
-		game.CoralBonusCoopEnabled = true
-	}()
-
-	testCases := []struct {
-		coopEnabled bool
-		matchType   model.MatchType
-	}{
-		// 0. Playoff match with co-op enabled.
-		{true, model.Playoff},
-
-		// 1. Playoff match with co-op disabled.
-		{false, model.Playoff},
-
-		// 2. Qualification match with co-op disabled.
-		{false, model.Qualification},
-	}
-
-	for i, tc := range testCases {
-		t.Run(
-			strconv.Itoa(i),
-			func(t *testing.T) {
-				arena := setupTestArena(t)
-				var plc FakePlc
-				plc.isEnabled = true
-				arena.Plc = &plc
-				game.CoralBonusCoopEnabled = tc.coopEnabled
-				arena.CurrentMatch.Type = tc.matchType
-
-				// Check that no inputs or outputs are active before the match starts.
-				assert.Equal(t, PreMatch, arena.MatchState)
-				plc.redProcessorCount = 5
-				plc.blueProcessorCount = 8
-				arena.Update()
-				redScore := &arena.RedRealtimeScore.CurrentScore
-				blueScore := &arena.BlueRealtimeScore.CurrentScore
-				assert.Equal(t, 0, redScore.ProcessorAlgae)
-				assert.Equal(t, 0, blueScore.ProcessorAlgae)
-				assert.Equal(t, [3]bool{false, false, false}, plc.redTrussLights)
-				assert.Equal(t, [3]bool{false, false, false}, plc.blueTrussLights)
-				plc.redProcessorCount = 0
-				plc.blueProcessorCount = 0
-
-				// Start the match.
-				arena.AllianceStations["R1"].Bypass = true
-				arena.AllianceStations["R2"].Bypass = true
-				arena.AllianceStations["R3"].Bypass = true
-				arena.AllianceStations["B1"].Bypass = true
-				arena.AllianceStations["B2"].Bypass = true
-				arena.AllianceStations["B3"].Bypass = true
-				arena.Update()
-				assert.Nil(t, arena.StartMatch())
-				arena.Update()
-				arena.MatchStartTime = time.Now().Add(-time.Duration(game.MatchTiming.WarmupDurationSec) * time.Second)
-				arena.Update()
-				assert.Equal(t, AutoPeriod, arena.MatchState)
```

Comment:
it seems like there's non-scoring functionality being tested here as well. should this be a more targeted removal of code?

Reply:
ok it looks like there are other cases that have these same asserts